### PR TITLE
Changed snake case naming to camelCase.

### PR DESCRIPTION
**Pull request by reikani**
Gets most of #11605.

Only missing pkg/system/stat.go, "Stat_t". Changes would reach into vendor files.

### CODE REVIEW

**Comment by LK4D4** [comment]
@reikani You need to rebase.

**Comment by reikani** [comment]
@LK4D4 Done, waiting on checks.

File: pkg/jsonlog/jsonlog_marshalling.go [hunk 1]
```diff
@@ -86,7 +86,7 @@ func (mj *JSONLog) MarshalJSONBuf(buf *bytes.Buffer) error {
 			buf.WriteString(`,`)
 		}
 		buf.WriteString(`"log":`)
-		ffjson_WriteJsonString(buf, mj.Log)
+		ffjsonWriteJSONString(buf, mj.Log)
```
Comment (LK4D4) [inline pkg/jsonlog/jsonlog_marshalling.go]: Can you exclude this fie, pls? it is autogenerated.

Comment (reikani) [inline pkg/jsonlog/jsonlog_marshalling.go]: The file's excluded, do the other pkg files need to be excluded as well?

**Comment by LK4D4** [comment]
LGTM

**Comment by cpuguy83** [comment]
LGTM

**Pull request merged**